### ISSUE TITLE
Rechenweg

### ISSUE THREAD

**Issue by pkunze** [Hartz-Arbeit/LohntDat#1]
Also ich bin nicht der Riesen Frontend Guru aber wenn mir jemand die Berechnung grob erklärt Steuere ich gerne eine Komponente und UnitTests für die Rechnung bei.

**Comment by IridiumWasTaken** [comment]
Sollten wir nicht vielleicht zuerst kurz über Features sprechen, die wir haben möchten? Bezüglich des Rechenwegs gibt's ja auch ein paar Dinge, wie z.B. zwingende/optionale Angaben, die zu klären wären. 
Ich fände es z.B. noch interessant zu sehen, wie lange es dauern würde, bis sich arbeiten nicht mehr lohnt, wenn die Inflation beim aktuellen Satz verharrt.

**Comment by pkunze** [comment]
Gute Idee. Siehe dazu #3. In diesem Issue würde ich beim Rechenweg bleiben um die Diskussionen nicht zu vermischen.

**Comment by pkunze** [comment]
Ein Gedanke noch dazu: um die erste Version (MVP) so schnell wie möglich auf die Beine zu stellen glaube ich es wäre sinnvoll von den ärmsten Arbeitnehmern auszugehen. Also solche Dinge wie Nebeneinkünfte/Mieterträge/Kapitalerträge und auch Vermögen (in der ersten Version) wegzulassen und davon auszugehen, dass der User nichts hat, außer seine Einkünfte als AN. Später kann und sollte man das hinzufügen. Aber die Leute die am wahrscheinlichsten als Ergebnis Hartz4 erhalten haben vermutlich kein Vermögen und keine weiteren Einkünfte.

**Comment by IridiumWasTaken** [comment]
Wo sammeln wir die nötigen Daten inkl. Quelle?

**Comment by pkunze** [comment]
@IridiumWasTaken bin mir nicht sicher, ob das richtig verstehe. Welche Daten genau meinst du?

**Comment by IridiumWasTaken** [comment]
@pkunze Genau, Freibeträge sowie zusätzliche Informationen wie z.B. Strom- und Gaspreise zur Ermittlung der weiteren Kosten (in diesem Fall Wohnen) sowie die maximale Unterstützung des Staates in diesen Bereichen. Weitere Daten wären z.B. durchschnittliche Pendelstrecken, Spritpreise etc.

**Comment by pkunze** [comment]
Okay, verstehe. Vermutlich kann man einiges hard-coden (zumindest für den Anfang) und anderes evtl. realTime aus APIs saugen. Ich versuche mich mal schlau zu machen, wie die Leute bei SchnellTestTest das gemacht haben, aber ich meine da liegen die Daten in JSON Files direkt im Repo bzw. der App.

**Comment by pkunze** [comment]
In #3 wurde ein gute Excel-Sheet hinterlegt, das ja auch ohne Datenhaltung in dem Sinne auskommt. Darauf könnte man aufsetzen. Siehe https://github.com/Hartz-Arbeit/LohntDat/issues/3#issuecomment-1207761136

**Comment by cr4yfish** [comment]
Hardcode sollte in diesem Fall eigentlich reichen.
> JSON Files direkt im Repo
+ 1

Änderungen müssen nur im Fall von Gesetzesänderungen geschehen - und die werden ja weit im Voraus angekündigt.

**Comment by Hartz-Arbeit** [comment]
Hier gibts ja so nen Rechner https://www.hartziv.org/hartz-iv-rechner/
Würde das noch weiter vereinfachen das die Leute auch mitmachen.
Am Ende als Ergebnis vielleicht den "effektiven Stundenlohn" anzeigen, Also was man mit der Netto Arbeit über den Satz verdient.
Das sollte die Leute dann recht schnell Triggern.

**Comment by jpv-os** [comment]
@Hartz-Arbeit Wunderbar - es ist gut, eine zweite Version des Rechners zum Vergleich zu haben. Das XSLX Sheet ist für uns deswegen besonders praktisch, weil der tatsächliche Rechenweg dort enthalten ist, den ich auf der verlinkten Seite nicht so klar nachlesen konnte. 

Dafür zeigt deine Version nochmal, dass man die Daten auch sehr viel schöner und kompakter abfragen kann. Die wesentlichen Kategorien, die man angeben muss, sind da sehr deutlich: Lebensumstände, Einkommen, Mehrbedarf, Kinder. Damit die App sinnvoll funktioniert, muss diese Eingabe ja so einfach und schnell wie möglich sein. Ich persönlich fänd's trotzdem cool, wenn wir zusätzlich zu der einfachen Version auch eine vollständige Berechnung anbieten können, vielleicht in einem Menü zum Aufklappen oder ähnlich.
